aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -19,6 +19,298 @@ The model classes you create are known as POCO classes (from **P**lain **O**ld *
 
 In this tutorial, you write the model classes first, and EF Core creates the database. An alternate approach not covered here is to generate model classes from an existing database. For information about that approach, see [ASP.NET Core - Existing Database](/ef/core/get-started/aspnetcore/existing-db).
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Add a data model class
+
+# [Visual Studio](#tab/visual-studio)
+
+Right-click the *Models* folder > **Add** > **Class**. Name the class **Movie**.
+
+[!INCLUDE [model 1b](~/includes/mvc-intro/model1b.md)]
+
+# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
+
+* Add a class to the *Models* folder named *Movie.cs*.
+
+[!INCLUDE [model 1b](~/includes/mvc-intro/model1b.md)]
+[!INCLUDE [model 2](~/includes/mvc-intro/model2.md)]
+
+---
+
+## Scaffold the movie model
+
+In this section, the movie model is scaffolded. That is, the scaffolding tool produces pages for Create, Read, Update, and Delete (CRUD) operations for the movie model.
```

Comment:
It's not really the model that is being scaffolded, it's the pages.  The model already exists and is the basis for scaffolding the pages.